hash/psx.xml

```diff
@@ -61498,6 +61893,48 @@ During gameplay, the commentary speech audio is playing too fast
 		</part>
 	</software>
 
+	<software name="kournikov">
+		<!--
+		http://redump.org/disc/5500
+		<rom name="Anna Kournikova's Smash Court Tennis (Europe) (Rev 1).cue" size="119" crc="8f6ccdab" sha1="99d1f1864e3bce4c8f1e2772a378c2e7d9f23e94"/>
+		<rom name="Anna Kournikova's Smash Court Tennis (Europe) (Rev 1).bin" size="379662192" crc="9b59eec7" sha1="fe27bedeb0160af682be2573733118d52e480202"/>
+		-->
+		<description>Anna Kournikova's Smash Court Tennis (Europe, rev. 1)</description>
+		<year>1999</year>
+		<publisher>Namco</publisher>
+		<info name="copy_protection" value="Error Detection and Correction (EDC)"/>
+		<info name="developer" value="Namco"/>
+		<info name="language" value="English" />
+		<info name="serial" value="SCES-01833#"/>
+		<sharedfeat name="compatibility" value="PAL-E"/>
+		<part name="cdrom" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="Anna Kournikova's Smash Court Tennis (Europe) (Rev 1)" sha1="300fc7a85c8f3a219a542a0e990ee50b811831b9"/>
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="kournikov_a" cloneof="kournikov">
```

Comment:
These romset ids selections are poor, I'd call it `smcourt2` / `smcourt2a` and the jp version `smcourt2j`